### DISCUSSION TITLE
Infinite scrolling, dev/log console example

### DISCUSSION THREAD

**Discussion by XeonG** [Unity-UI-Extensions/com.unity.uiextensions#447] [General]
So I have a dev console.. new items get added like logs in a verticle scrollbar to the bottom... if the scrollbar is at the bottom new items should jumped too and in focus otherwise not.. my current implemention just uses the default ugui scrollbar stuff, instantiating new prefabs (the prefabs height size can vary depending on text content.. this all works fine with content sizing ...  mainly adding new items in a performant way is what I really need now as ton entries coming in does just tank it, being able to remove an item would be nice.

Can't seem to find a demo example that is close to that demostrating adding new items without all the fluff like all the fancyscroll ui's or just the assumed fixed size of items, dunno isn't a example that is close to what I'm trying to do. 

Just seem to have endless problem trying to find something as its bit of a different implementation with cell prefab.. also battle with prefab sizing being different depending on the cell text content going into it... 

Anyway be great to get some basic example of this

**Comment by XeonG** [comment]
https://github.com/Brian-Jiang/UnlimitedScrollUI/issues/23  ...just another ugui scrollbar that I haven't tried yet.. .seems to be problems with all of them, is there nothing that actually does this stuff out of the box right?

**Comment by SimonDarksideJ** [comment]
Thanks for the report @XeonG , only just spotted this.

So am I understanding the issue is that the infinite scroll controls do not have the ability to add items to the "current" bottom of the list?
